docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,171 @@
+
+### <a name="AbstractJdbcPOJOOutputOperatorProps"></a>Properties of AbstractJdbcPOJOOutputOperator
+Several properties are available to configure the behavior of this operator and they are summarized in the table below.
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *tablename* | Name of the table where data is to be inserted | String | Yes | N/A |
+| *fieldInfos*| JdbcFieldInfo maps a store column to a POJO field name| List | Yes | N/A |
+Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.tablename</name>
+  <value>tablename</value>
+</property>
+
+<property>
+    <name>dt.operator.{OperatorName}.fieldInfosItem[0]</name>
+    <value>
+      {
+        "sqlType": 0,
+        "columnName":"ID",
+        "pojoFieldExpression": "id",
+        "type":"INTEGER"
+      }
+    </value>
+  </property>
+
+  <property>
+    <name>dt.operator.{OperatorName}.fieldInfosItem[1]</name>
+    <value>
+      {
+        "sqlType": 4,
+        "columnName":"NAME",
+        "pojoFieldExpression": "name",
+        "type":"STRING"
+      }
+    </value>
+  </property>
+```
+
+
+## Platform Attributes that influences operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *input.TUPLE_CLASS* | TUPLE_CLASS attribute on input port which tells operator the class of POJO which is being received  | Class| Yes |
+Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.port.input.attr.TUPLE_CLASS</name>    
+    <value>com.example.mydtapp.PojoEvent</value>
+</property>
+```
+
+One of the concrete implementation is provided in Malhar as [JdbcPOJOInsertOutputOperator](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPOJOInsertOutputOperator.java).  The incoming tuples will be inserted in the table using PreparedStatement of the base class,  which is formed in activate() method of this operator.
+As shown in the UML diagram below, this class has several superclasses, so its behavior can be configured via properties of those classes as described below.
+![JdbcPOJOInsertOutputOperator.png](images/jdbcoutput/operatorsClassDiagram.png)
+
```

Comment:
The diagram shows the hierarchy ending at `AbstractPassThruTransactionableStoreOutputOperator` but that class extends `AbstractTransactionableStoreOutputOperator` which again has a base class and so forth; please show the complete hierarchy.